### PR TITLE
Change pickle to cloudpickle for keras.multiprocessing backend

### PR DESCRIPTION
**Pull request by az2181036**

### CODE REVIEW

**Comment by TheaperDeng** [comment]
working on `pickle` vs `cloudpickle` as well (for torch), could I ask what's the advantage of `cloudpickle`?

**Comment by yangw1234** [comment]
> working on `pickle` vs `cloudpickle` as well (for torch), could I ask what's the advantage of `cloudpickle`?

image

**Comment by az2181036** [comment]
> working on `pickle` vs `cloudpickle` as well (for torch), could I ask what's the advantage of `cloudpickle`?

Just change the codes about Keras, and I got an error about can't load CustomCallbacks ( inherited from keras.callbacks.Callback) this morning.

**Comment by TheaperDeng** [comment]
very helpful, thx!

**Comment by yangw1234** [comment]
LGTM

**Comment by az2181036** [comment]
http://10.112.231.51:18888/job/ZOO-PR-NanoTests/357/
